bundle/bundle_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/deislabs/cnab-go/bundle/definition"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Nit: I think aliasing this import is no-op, as golang already takes the part before the first point as import name.